javascript/modules/client/test/integration/client-erc20.test.ts

```diff
@@ -89,88 +112,92 @@ describe("Client", () => {
+      expect(client.graphql.getClient()).toBeInstanceOf(GraphQLClient);
 
-      const web3Status = await client.web3.isUp();
-      expect(web3Status).toEqual(false);
-    });
-    it("Should create a client, fail and shift to a working endpoint", async () => {
-      contextParams.web3Providers = web3endpoints.failing.concat(
-        web3endpoints.working,
-      );
-      const context = new ContextErc20(contextParams);
-      const client = new ClientErc20(context);
-
-      await client
-        .web3.isUp()
-        .then((isUp) => {
-          expect(isUp).toEqual(false);
-          client.web3.shiftProvider();
-
-          return client.web3.isUp();
-        })
-        .then((isUp) => {
-          expect(isUp).toEqual(true);
-        });
+      // Web3
+      const web3status = await client.web3.isUp();
+      expect(web3status).toEqual(false);
+      // IPFS
+      const ipfsStatus = await client.ipfs.isUp();
+      expect(ipfsStatus).toEqual(false);
+      // GraqphQl
+      const graphqlStatus = await client.graphql.isUp();
+      expect(graphqlStatus).toEqual(false);
     });
   });
-
   describe("Proposal Creation", () => {
-    test.todo("Should estimate gas fees for creating a new proposal");
-    // it("Should estimate gas fees for creating a new proposal", async () => {
-    //   const context = new ContextErc20(contextParamsLocalChain);
-    //   const client = new ClientErc20(context);
-
-    //   const proposalParams: ICreateProposalParams = {
-    //     metadataUri: "ipfs://",
-    //     actions: [],
-    //     creatorVote: VoteOptions.YEA,
-    //     startDate: new Date(),
-    //     endDate: new Date(),
-    //     executeIfPassed: true,
-    //   };
-
-    //   const gasFeesEstimation = await client.estimation.createProposal(
-    //     proposalParams,
-    //   );
-
-    //   expect(typeof gasFeesEstimation).toEqual("object");
-    //   expect(typeof gasFeesEstimation.average).toEqual("bigint");
-    //   expect(typeof gasFeesEstimation.max).toEqual("bigint");
-    //   expect(typeof gasFeesEstimation.max).toBeGreaterThan(BigInt(0));
-    //   expect(gasFeesEstimation.max).toBeGreaterThan(gasFeesEstimation.average);
-    // });
-    it("Should create a proposal locally", async () => {
-      const context = new ContextErc20(contextParamsLocalChain);
-      const client = new ClientErc20(context);
+    it("Should estimate the gas fees for creating a new proposal", async () => {
+      const context = new ContextErc20(contextParamsLocalChain)
+      const client = new ClientErc20(context)
 
       const proposalParams: ICreateProposalParams = {
         metadataUri: "ipfs://",
-        actions: [
-          client.encoding.withdrawAction({
-            recipientAddress: "0x9a16078c911afAb4CE4B7d261A67F8DF99fAd877",
-            amount: BigInt(10),
-            reference: "Test",
-          }),
-        ],
-        creatorVote: VoteOptions.NAY,
+        actions: [],
+        creatorVote: VoteOptions.YEA,
```

Comment:
Isn't VotingOptions already using `YES/NO`  as an enum value?
Maybe I'm confused by the change on the smart contracts

Reply:
I wasn't, I updated All interfaces to use `yes/no`